jbpm-services/jbpm-kie-services/pom.xml

```diff
@@ -145,6 +145,11 @@
       <artifactId>reflections</artifactId>
     </dependency>
     
+	<dependency>
+	  <groupId>commons-lang</groupId>
+      <artifactId>commons-lang</artifactId>
+    </dependency>
```

Comment:
@mswiderski ok, I will get to it next week ... but if the discussion around that took too much time, I would like to see at least this file being updated [1], so we could create own KIE extension with the form service.

Thanks

[1] https://github.com/droolsjbpm/droolsjbpm-integration/pull/229/files#diff-14503dabad0998300652edd66e5b2052R269